### PR TITLE
fix(ci): only build the required derivation

### PR DESCRIPTION
**Pull request by Pacman99**
Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, PNG optimization, etc. are updated
- PR
    - [ ] (For external contributions) Corresponding issue exists and is linked in the description
    - [ ] Targeting master unless this is a cherry-pick backport
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reviewer requested

### CODE REVIEW

**Review by michaelpj — APPROVED** [review]
Looks fine! We could simplify it a bit now we're only building one job, but I guess leaving the rest in there so that it works if we add more stuff later is fine.

File: nix/cells/cloud/pipelines.nix [hunk 1]
```diff
@@ -35,9 +35,11 @@ let
       };
     };
 
-  # The 'required' job masks everything else, which isn't helpful, so we rmeove it
-  # for cicero
-  ciTasks = builtins.removeAttrs
+  # Only build the required job, other jobs might change and become unavailable
+  # This is necessary because cicero only evaluates the task list on master,
+  # instead of the current branch
+  ciJobsToBuild = lib.getAttrs [ "required" ] ciJobs;
```
[the remark this answers is not in the document]
Comment (michaelpj) [reply nix/cells/cloud/pipelines.nix:41]: Whatever, it's fine.

**Comment by michaelpj** [comment]
Stupid zfs bug

File: nix/cells/cloud/pipelines.nix (hunk 1 above)
[the remark this answers is not in the document]
Comment (Pacman99) [reply nix/cells/cloud/pipelines.nix:41]: :facepalm: I think I just forgot that `inherit` existed

**Pull request merged**